addons/a11y/package.json

```diff
@@ -20,8 +20,16 @@
     "directory": "addons/a11y"
   },
   "license": "MIT",
-  "main": "dist/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
this adds ESM exports to these packages, webpack can find these different entrypoints

Reply:
This is kinda backwards incompatible, any imports directly to /dist/* will fail now.